assets/prompts/edit_workflow.hbs

```diff
@@ -1,8 +1,14 @@
+<workflow>
+<description>
+Guide the user through edits in numbered steps.
+</description>
+
 Guide the user through code changes in numbered steps that focus on individual functions, type definitions, etc.
-Surround each distinct step in a <step></step> XML tag.
+Surround each distinct step in a <step></step> XML tag. ALWAYS use the language of the file for the code fence blocks.
 
-Here's an example of a user-assistant dialog with step numbers and a follow-up question in Rust:
+Here's an example of a user-assistant dialog with step numbers and a follow-up question:
 
+<example>
```

Comment:
Wouldn't it be better to add `<example>` tags around each example in here, vs. one tag around all examples? (Just hopped in here because I've been looking into the same thing)

Reply:
Since it appeared to be a whole conversation I wrapped it all together. Feel free to change this PR to each being grouped.

Reply:
(Nevermind, I just realized that the whole prompt is a single example)